tools/scripts/track-pr

```diff
@@ -43,6 +45,19 @@ NEEDS_TESTING_STATUS = "Needs testing"
 FIX_OPEN_STATUS = "Fix (open)"
 FIX_CONFLICT_STATUS = "Fix (conflict)"
 
+GITHUB_TOKEN = os.environ["GITHUB_TOKEN"]
+
+
+def run(*args, check=True, quiet=False, **kwargs):
+    kwargs = dict(kwargs)
+    if not quiet:
+        print("\n================ running: \x1b[36m{}\x1b[m".format(" ".join(args)))
```

Comment:
<obligatory comment about the miracle of f-strings here>

Reply:
I tend to like f-strings less when the thing inside is more than a variable name, and especially if it contains quotes. Though in this case I was thinking lately that I should just print the array directly to avoid ambiguity anyway, so I guess I'll just put in `{args}`.